news/1042.feature

```diff
@@ -0,0 +1,22 @@
+A new service endpoint and expand is available, a Navigation Portlet exposed as
+``@navportlet``. It uses the same semantics as the classic Plone navigation
+portlet, largely through reusing the same code. Instead of storing the
+"portlet" configuration in a portlet assignment storage, you can pass these as
+parameters to the service or expand component.
+
+You can provide these as parameters:
+
+- ``name`` - The title of the navigation tree.
+- ``root_path`` - Root node path, can be "frontend path", derived from router
```

Comment:
What does `can be "frontend path", derived from router`  mean? This does not make any sense to me.

Reply:
Meaning "window location". The "physical path" in Zope requires rooting the path in the ZODB root, so you'd have to know the name of the Plone website. In contrast, the path accepted here can consider its root as the Plone site root

Reply:
ok :) well, docs need to be written anyway.